### PR TITLE
Upload assets s3

### PR DESCRIPTION
**Pull request by tejunior**

### CODE REVIEW

File: .github/workflows/test_release_linux.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  DOCKER_PUBLISH: false
+
+jobs:
+  publishing-to-s3:
+    name: Publish linux artifacts into s3 test bucket
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Publish assets to S3 action
+        uses: newrelic/infrastructure-publish-action@v1.0.7
```
Comment (varas) [inline .github/workflows/test_release_linux.yml:24]: v1 pls

File: .github/workflows/prerelease_linux.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: Prerelease Linux pipeline
+
+on:
+  push:
+    branches: tempToBeReplacedByMain
+
+env:
+  TAG: 1.7.4
```
Comment (brushknight) [inline .github/workflows/prerelease_linux.yml:8]: Tag should be from release event

File: .github/workflows/prerelease_linux.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  DOCKER_PUBLISH: false
+
+jobs:
+  publishing-to-s3:
+    name: Publish linux artifacts into s3 test bucket
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Publish assets to S3 action
+        uses: newrelic/infrastructure-publish-action@v1.0.7
```
Comment (varas) [inline .github/workflows/prerelease_linux.yml:23]: v1 pls

File: .github/workflows/prerelease_linux.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+name: Prerelease Linux pipeline
+
+on:
+  push:
```
Comment (varas) [inline .github/workflows/prerelease_linux.yml:4]: on prerelease pls

File: .github/workflows/prerelease_linux.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    branches: tempToBeReplacedByMain
+
+env:
+  TAG: 1.7.4
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  AWS_S3_BUCKET_NAME: "nr-downloads-ohai-staging"
+  AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock-staging"
+  AWS_REGION: "us-east-1"
+  DOCKER_PUBLISH: false
```
Comment (brushknight) [inline .github/workflows/prerelease_linux.yml:14]: not needed in this pipeline

File: .github/workflows/test_release_linux.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+name: Logging Test Linux pipeline
+
+on:
+  push:
+    branches:
```
Comment (varas) [inline .github/workflows/test_release_linux.yml:5]: I guess you'd like a manual dispatch instead for this one

File: .github/workflows/prerelease_linux.yml (hunk 2 above)
Comment (varas) [inline .github/workflows/prerelease_linux.yml:8]: `TAG: ${{ github.event.release.tag_name }}`

File: .github/workflows/prerelease_linux.yml (hunk 4 above)
Comment (varas) [reply .github/workflows/prerelease_linux.yml:4]: see https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L3-L8

File: .github/workflows/test_release_linux.yml (hunk 6 above)
Comment (varas) [reply .github/workflows/test_release_linux.yml:5]: see https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L3-L8

**Comment by brushknight** [comment]
in the schema file you will need to replace part of the src_repo like here https://github.com/newrelic/infrastructure-agent/blob/master/build/upload-schema-linux-deb.yml#L9

**Review by varas — CHANGES_REQUESTED** [review]
pls add the new `access_point_host` param, see:
- release https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/release.yml#L61
- pre-release https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L160

File: .github/workflows/test_release_linux.yml (hunk 6 above)
Comment (brushknight) [reply .github/workflows/test_release_linux.yml:5]: I think this is for test, so probably they want to have them always on push 🤔

File: .github/workflows/test_release_linux.yml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+      - '*'
+      - '!master'
+env:
+  TAG: 1.7.4
+  GPG_PASSPHRASE: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+  GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
+  AWS_S3_BUCKET_NAME: "nr-downloads-logging-testing"
```
Comment (brushknight) [inline .github/workflows/test_release_linux.yml:12]: should point to staging, ref: https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_linux.yml#L21-L22

File: .github/workflows/test_release_linux.yml (hunk 6 above)
Comment (brushknight) [reply .github/workflows/test_release_linux.yml:5]: ah, got it, it is a test release.

Comment (tejunior) [reply .github/workflows/test_release_linux.yml:5]: Yes, right now it's running always we push...   I will validate with the team if they prefer keep as is, run when we create a PR or even manual dispatch as you suggested

File: .github/workflows/test_release_linux.yml [hunk 8]
```diff
@@ -0,0 +1,46 @@
+name: Logging Test Linux pipeline
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+env:
+  TAG: ${{ github.event.release.tag_name }}
```
Comment (varas) [inline .github/workflows/test_release_linux.yml:9]: this is not triggered by a release event

Comment (tejunior) [reply .github/workflows/test_release_linux.yml]: True, removed from the pipeline!

File: schemas/fb-linux.yml [hunk 9]
```diff
@@ -5,7 +5,7 @@
     - arm64
   uploads:
     - type: apt
-      src_repo: "http://download.newrelic.com/infrastructure_agent/linux/apt"
+      src_repo: "{access_point_host}/infrastructure_agent/linux/apt"
```
Comment (varas) [inline schemas/fb-linux.yml:8]: 👏

**Review by varas — APPROVED** [review]

**Pull request merged**